storybook/src/stories/Global/GlobalUI.stories.tsx

```diff
@@ -487,15 +489,39 @@ export const _GlobalUI = () => {
   ])
   const notificationsHistory = object("Notifications History", allNotifications);
 
+  const userConfig = object("User Notes", {
+    items: [
+      {
+        icon: 'Camera',
+        title: 'Camera Device',
+        subTitle: 'Building A',
+        notes: `This device is on fourth floor. connected with two USB devices.`,
+      },
+      {
+        icon: 'Information',
+        title: 'Nuc-1',
+        subTitle: 'Building B',
+        notes: `This device is on third floor. connected with two USB devices.`,
```

Comment:
`This device is on third floor. connected with two USB devices.`  --> `This is a GPU machine having 30 camera support`

Reply:
Done